backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/impl/DocumentSequenceDAO.java

```diff
@@ -197,7 +199,7 @@ public DocTypeSequenceMap retrieveDocTypeSequenceMap(final I_C_DocType docType)
 		return retrieveDocTypeSequenceMap(ctx, docTypeId);
 	}
 
-	@Cached(cacheName = I_C_DocType_Sequence.Table_Name + "#by#" + I_C_DocType_Sequence.COLUMNNAME_C_DocType_ID)
+	@Cached(cacheName = I_C_DocType_Sequence.Table_Name + "#and#" + I_AD_DocType_CountryBased.Table_Name + "#by#" + I_C_DocType_Sequence.COLUMNNAME_C_DocType_ID)
```

Comment:
did you make sure that cache.invalidation actually works that way?
I mean, will be the cache invalidated if either `C_DocType_Sequence` or `AD_DocType_CountryBased` are changed?
If not, or if you are unsure, please create the cache "explicitly" via `CCache.<K, V> builder()`